README.md

```diff
@@ -68,6 +68,18 @@ You can shorten multiple `--from` parameters using commas without spaces
 npx update-badge X --from bahmutov/repo1,bahmutov/repo2,bahmutov/repo3
 ```
 
+You can use custom file name instead README.md using `--fileName` parameter
+
+```
+npx update-badge --fileName info.md
+```
+
+You can use custom file path using `--filePath` parameter
+
+```
+npx update-badge --filePath ./folder1/folder2
```

Comment:
Hmm, I find this confusing (splitting the argument into two like this). For example, if I want to update a badge in file `./docs/index.md` I have to use both arguments? I would prefer using just one explicit one, something like
```
npx update-badge --file ./docs/index.md
```

Reply:
So, If you want to change the file name or path, you have to provide them both together?